### ISSUE TITLE
When tabbing around buttons, it is not visible that they get focus

### ISSUE THREAD

**Issue by mitar** [peerlibrary/peerlibrary#410]
When tabbing around buttons, it is not visible that they get focus. (In meta menus, link dialog, etc.)
